x-pack/plugins/ml/public/application/routing/routes/explorer.tsx

```diff
@@ -109,9 +112,26 @@ const ExplorerUrlStateManager: FC<ExplorerUrlStateManagerProps> = ({ jobsWithTim
     }
   }, [globalState?.time?.from, globalState?.time?.to]);
 
+  const getJobsWithStoppedPartitions = async (selectedJobIds: string[]) => {
+    try {
+      const fetchedStoppedPartitions = await ml.results.getStoppedPartitions(
+        selectedJobIds,
+        JOB_ID
+      );
+      if (fetchedStoppedPartitions.jobs.length > 0) {
+        setStoppedPartitions(fetchedStoppedPartitions.jobs);
+      } else {
+        setStoppedPartitions(undefined);
+      }
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
```

Comment:
I think for this scenario it's okay to fail silently since it's only there to give a warning/info message if one of the partition has stopped. But if we think it's important for the user to know, I can add an error message.